Upstream: Installs gawk in /bin.
Source: philantrop@exherbo.org
Summary: Find out where gawk resides and use it whereever it is.

diff -Naur baselayout-1.12.12.orig/sbin/depscan.sh baselayout-1.12.12/sbin/depscan.sh
--- baselayout-1.12.12.orig/sbin/depscan.sh	2008-01-01 19:23:37.000000000 +0100
+++ baselayout-1.12.12/sbin/depscan.sh	2008-05-23 01:05:11.000000000 +0200
@@ -6,6 +6,8 @@
 	trap ":" INT QUIT TSTP
 fi
 
+_GAWK=$(type -p gawk)
+
 argv0=${0##*/}
 source /etc/init.d/functions.sh || {
 	echo "${argv0}: Could not source /etc/init.d/functions.sh!" 1>&2
@@ -133,13 +135,13 @@
 
 cd /etc/init.d
 
-/bin/gawk \
+${_GAWK} \
 	-f /lib/rcscripts/awk/functions.awk \
 	-f /lib/rcscripts/awk/cachedepends.awk || \
 	retval=1
 
 bash "${mysvcdir}/depcache" | \
-/bin/gawk \
+${_GAWK} \
 	-f /lib/rcscripts/awk/functions.awk \
 	-f /lib/rcscripts/awk/gendepends.awk || \
 	retval=1
diff -Naur baselayout-1.12.12.orig/sbin/env-update.sh baselayout-1.12.12/sbin/env-update.sh
--- baselayout-1.12.12.orig/sbin/env-update.sh	2007-10-24 00:59:27.000000000 +0200
+++ baselayout-1.12.12/sbin/env-update.sh	2008-05-23 01:05:22.000000000 +0200
@@ -9,6 +9,8 @@
 	exit 1
 fi
 
+_GAWK=$(type -p gawk)
+
 usage() {
 echo "usage: env-update.sh
 
@@ -30,7 +32,7 @@
 if [[ $# != "0" ]] ; then
 	usage
 else
-	/bin/gawk \
+	${_GAWK} \
 		-f /lib/rcscripts/awk/functions.awk \
 		-f /lib/rcscripts/awk/genenviron.awk
 fi
